vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -11134,23 +11164,28 @@ quantitykind:MechanicalEnergy
 .
 quantitykind:MechanicalImpedance
   a qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A0E-2L2I0M1H0T-3D0 ;
```

Comment:
I'm wondering why it isn't Mass/Time.

Reply:
Not a narrower version of quantitykind:Impedance, which is electrical.

Reply:
Then that is my mistake. I assumed that it was the same units as Impedance and took the dimension vector from Impedance.

Reply:
Agreed, Mass/Time.

Reply:
Fixed